sabnzbd/utils/checkdir.py

```diff
@@ -80,35 +74,37 @@ def isFAT(check_dir):
             # Then: device => filesystem type
             server:~ sander$ mount | grep /dev/disk9s1
             /dev/disk9s1 on /Volumes/CARTUNES (msdos, local, nodev, nosuid, noowners)
-
-
             """
-            dfcmd = "df " + check_dir
-            for thisline in getcmdoutput(dfcmd):
+            for thisline in getcmdoutput(["df", check_dir]):
                 if thisline.find("/") == 0:
-                    if debug:
-                        print(thisline)
                     # Starts with /, so a real, local device
                     device = thisline.split()[0]
-                    mountcmd = "mount | grep " + device
-                    mountoutput = os.popen(mountcmd).readline().strip()
-                    if debug:
-                        print(mountoutput)
+
+                    # Run the equivalent of "mount | grep $device"
+                    p_mount = subprocess.Popen(["mount"], stdout=subprocess.PIPE)
```

Comment:
This cannot be done through ` getcmdoutput`?
Like `getcmdoutput(["mount","|","grep","device"])?

Reply:
Oh I see that stackoverflow says _no_ 😔